### DISCUSSION TITLE
Add Read Later options

### DISCUSSION THREAD

**Discussion by anubhavbhatt** [dailydotdev/daily#780] [Feature Requests]
Sometimes, we have good articles so we read the TLDR, but no time for full reading the articles. So please provide an option to add Read Later button for that articles. So when we have time in weekends we read full article later. As the articles gone once we review.
I know bookmark option is there but Bookmark is for keeping a very good article for future use.

**Comment by idoshamun** [comment]
Hi @anubhavbhatt,

Thanks a lot for the suggestion. We have a discussion about it already #364.
Feel free to join.
